src/bitcoind_client.rs

```diff
@@ -184,7 +184,16 @@ impl BitcoindClient {
 		let mut rpc = self.bitcoind_rpc_client.lock().await;
 
 		let raw_tx_json = serde_json::json!(raw_tx.0);
-		rpc.call_method("fundrawtransaction", &[raw_tx_json]).await.unwrap()
+		let options = serde_json::json!({
+			"fee_rate": self.get_est_sat_per_1000_weight(ConfirmationTarget::Normal) as f64 / 250.0,
```

Comment:
Maybe write out the unit conversion math (`* 4 / 1000`) for guys like me figuring out the math. 😛